portal/urls.py

```diff
@@ -127,6 +127,7 @@
     url(r'^play/details/$', student_details, name='student_details'),
     url(r'^play/account/$', student_edit_account, name='student_edit_account'),
     url(r'^play/join/$', student_join_organisation, name='student_join_organisation'),
+    url(r'^play/aimmo/$', TemplateView.as_view(template_name='portal/play_aimmo.html'), name='play_aimmo'),
```

Comment:
should rapid router be `/play/rapid-router` and then `/play` can be re-routed to that?